Examples/Python/ellipsoid.py

```diff
@@ -28,7 +28,7 @@ def Run_Pipeline(args):
     if int(args.interactive) != 0:
         input("Press Enter to continue")
     # Get data
-    datasetName = "ellipsoid-v0"
+    datasetName = "ellipsoid-v2"
```

Comment:
should we move to ellipsoid and ellispoid-aligned rather than v2 and v0?

Reply:
We could make ellipsoid-v2 just ellipsoid since it is a different dataset altogether than ellipsoid-v0. I think we should have a bigger discussion about the versioning because it could get confusing.

Reply:
Till Kitware have a better solution for versioning, we will stick to -vN suffix for datasets. What I meant is rather than having ellipsoid-v0 that implicitly refer to groomed ellispsoids and ellpsoid-v2 that implicitly refer to ungroomed ones, we should consider having ellipsoid-v1 and ellispoid-aligned-v1 (that contain regenerated ellispoids without misalignment using the shape cohort generator). This way it is more explicit and consistent.

Reply:
I see, so should I just rename ellipsoid-v2 to ellipsoid-v1 and ellipsoid_aligned-v0 to ellipsoid_aligned-v1?

Reply:
Yes. I assume ellispoid_aligned-v1 has data generated from the cohort generator.

Reply:
Yes that's right

Reply:
Addressed b5152c3